auth-web/src/views/pay/eft/ShortNameDetailsView.vue

```diff
@@ -0,0 +1,68 @@
+<template>
+  <v-container
+    id="shortname-details"
+    class="view-container"
+  >
+    <div class="view-header flex-column">
+      <h1 class="view-header__title">
+        Payment Details
+      </h1>
+      <p class="mt-3 mb-0">
+        Review and verify payment details
+      </p>
+    </div>
+    <ShortNameTransactions :shortName="state.result" />
+    <br> <br>
+    <ShortNameAccountLinkage :shortName="state.result" />
+  </v-container>
+</template>
+<script lang="ts">
+import { PropType, defineComponent, onMounted, reactive } from '@vue/composition-api'
+import PaymentService from '@/services/payment.services'
+import ShortNameAccountLinkage from '@/components/pay/eft/ShortNameAccountLink.vue'
+import ShortNameTransactions from '@/components/pay/eft/ShortNameTransactions.vue'
+
+export default defineComponent({
+  name: 'ShortNameMappingView',
+  components: { ShortNameAccountLinkage, ShortNameTransactions },
+  props: {
+    shortNameId: {
+      type: String as PropType<string>,
+      default: null
+    }
+  },
+  setup (props) {
+    const state = reactive({
+      result: {}
```

Comment:
we may have used this before, but probably a better name than `result` available